.github/scripts/pod_spell_check.t

```diff
@@ -8,7 +8,7 @@ if (!@ARGV) {
     die "Missing perl file to check.";
 }
 
```

Comment:
As the "pod_spell_check." script relates to perl, could you adapt the "CODE_OWNER" file and add it to the owners-perl's scope ?